src/run/install/package-json.js

```diff
@@ -6,8 +6,12 @@ export default function (projectPath) {
   const packagePath = path.join(projectPath, 'package.json')
   const packageJSON = json.read(packagePath)
 
-  json.write(packagePath, {
-    ...packageJSON,
-    scripts: updateNpmScripts(packageJSON.scripts)
-  })
+  const newScripts = updateNpmScripts(packageJSON.scripts)
+  const scriptsNeedsToBeUpdated = JSON.stringify(newScripts) !== JSON.stringify(packageJSON.scripts)
```

Comment:
Well, _actually_ (sorry) JSON.stringify is not deterministic, so it's not trustworthy to do this :( . We can use https://github.com/substack/node-deep-equal instead :D